src/pages/reference/handlers/graphql.md

```diff
@@ -0,0 +1,109 @@
+---
+id: graphql
```

Comment:
Global: Delete the `id` and `sidebar_label` frontmatter lines unless they are used on the Devsite. If they are being used, let me know what they're doing. Adding descriptions can wait till after beta.